src/mlpack/methods/xgboost/loss_functions/sse_loss.hpp

```diff
@@ -89,8 +89,8 @@ class SSELoss
     gradients = (input.row(1) - input.row(0)).t();
     hessians = arma::vec(input.n_cols, arma::fill::ones);
 
-    return std::pow(ApplyL1(arma::accu(gradients)), 2) /
-        (arma::accu(hessians) + lambda);
+    return std::pow(ApplyL1(accu(gradients)), 2) /
+        (accu(hessians) + lambda);
```

Comment:
```suggestion
    return std::pow(ApplyL1(accu(gradients)), 2) / (accu(hessians) + lambda);
```
Fits on one line now :smile: